### ISSUE TITLE
Accessibility - iOS - Spanish - VoiceOver Does not Read Individual Steps on Main Steps Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#105]
While using VoiceOver to read the individual steps on the main Steps screen for Spanish, they are not getting read.
